API.md

```diff
@@ -2001,6 +2001,30 @@ This function is called when the state of the current document's undo/redo stack
 />
 ```
 
+## Wonday Compatibility API 
```

Comment:
i thought we split wonday to a different section, perhaps this PR should be drafted until the other PR is done